packages/components/slider/src/SliderThumb.styles.ts

```diff
@@ -0,0 +1,31 @@
+import { cva, type VariantProps } from 'class-variance-authority'
+
+export const thumbVariants = cva(
+  [
+    'block h-xl w-xl rounded-full cursor-pointer',
```

Comment:
```suggestion
    'block h-sz-24 w-sz-24 rounded-full cursor-pointer',
```

The reason for this suggestion ⬇️ 

Let's say someone has a theme with a customized value for their `spacing.xl` token, it would alter the styling of this thumb, and we don't want that imho